src/components/parts/AddToCartButton.vue

```diff
@@ -1,6 +1,6 @@
 <script>
 export default {
-  emits: ["hide-add-to-cart", "add-state-for-image"],
+  emits: ['hide-add-to-cart'],
```

Comment:
Rename this event to `toggle-active-product` because is more self explanatory.
You might need to use the inverse of the boolean where you listen for the change.